.github/workflows/functional.yml

```
@@ -71,7 +71,8 @@ jobs:
           RSTUF_STORAGE_BACKEND: LocalStorage
           RSTUF_KEYVAULT_BACKEND: LocalKeyVault
           RSTUF_LOCAL_STORAGE_BACKEND_PATH: /var/opt/repository-service-tuf/storage
-          RSTUF_LOCAL_KEYVAULT_PATH: /var/opt/repository-service-tuf/keystorage
+          RSTUF_LOCAL_KEYVAULT_PATH: /var/opt/repository-service-tuf/keystorage/online.key
```

Are you sure the path shouldn't be: 
```suggestion
          RSTUF_LOCAL_KEYVAULT_PATH: /var/opt/repository-service-tuf/tests/files/keystorage/online.key
```

Same for line 91.

---

It is not correct. We create an empty volume without a subfolder on lines 69 and 86 (using the same volume, rstuf-keystorage).
Later in the step Deploy online.key to Docker Volume, we copy the online.key in the root folder of this volume, which is /var/opt/repository-service-tuf/keystorage/ for workers 1 and 2.